src/App.tsx

```diff
@@ -1,24 +1,166 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { addTodos, deleteTodo, getTodos } from './api/todos';
+import { Error } from './components/Error/Error';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { Todolist } from './components/Todolist/Todolist';
+import { ErrorMessage } from './types/ErrorMessage';
+import { TodoStatus } from './types/TodoStatus';
 
-const USER_ID = 0;
+const USER_ID = 10890;
 
 export const App: React.FC = () => {
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<TodoStatus>(TodoStatus.all);
+  const [isError, setIsError] = useState<ErrorMessage>(ErrorMessage.noError);
```

Comment:
naming  with 'is' prepix is common for boolean data types.

Reply:
@growingananas Hey, thanks for the good point)
The variable has been renamed.